### ISSUE TITLE
Bots could use armories through transparent walls

### ISSUE THREAD

**Issue by necessarily-equal** [Unvanquished/Unvanquished#2294]
Example from mission dretch:

![image](https://user-images.githubusercontent.com/59283660/199691780-b61c96cb-2497-4290-8e6e-2dcbc3023f8e.png)

Currently bots will not be able to use it because they think they can't walk to it. Fixing this seems not so hard and fairly hard at the same time because one would have to find a suitable navmesh point, that is both accessible by the bot and (ideally) where the bot could see the armory so tha people don't feel like the bot is cheating.

**Comment by sweet235** [comment]
I'd rather have my bots cheaty than naked.

**Comment by necessarily-equal** [comment]
I mean it's not exactly cheaty because humans *could* do it too, but it's not intuitive and it looks bad.

**Comment by cu-kai** [comment]
Bots could be made to use /buy when in range of the armoury in the same way real players can. In this specific instance: this armoury can be `+USE`'d through the glass and requires no special binds to use.

**Comment by bmorel** [comment]
The problem is, bots won't be able to find a path to armories behind glasses.

**Comment by DolceTriade** [comment]
Imo, bots should be able to use the armory in any way a human could (ie, even through a non-transparent wall).

**Comment by bmorel** [comment]
I agree bots should have same abilities as humans, but this is still technically complex to solve, as there is no way to find a path to an enclosed building.
